test/unit_tests/k1d_unit_test.jl

```diff
@@ -65,14 +65,32 @@ TT.@testset "Initialise aux" begin
     coord = CC.Fields.coordinate_field(space)
     ip = map(coord -> _ip, coord)
 
-    TT.@test_throws Exception K1D.initialise_aux(FT, ip, params..., 0.0, 0.0, face_space, no_precip)
-    TT.@test_throws Exception K1D.initialise_aux(FT, ip, params..., 0.0, 0.0, face_space, CO.EquilibriumMoisture())
-    TT.@test_throws Exception K1D.initialise_aux(FT, ip, params..., 0.0, 0.0, face_space, CO.NonEquilibriumMoisture())
+    TT.@test_throws Exception K1D.initialise_aux(FT, ip, params..., 0.0, 0.0, face_space, no_precip, "ShipwayHill")
```

Comment:
It would be good to replace some of the `ShipwayHill` with `Jouan` options. So that we have good test coverage on both of them